python/python/lancedb/db.py

```diff
@@ -48,6 +55,12 @@ class EnforceOverrides:
     from .io import StorageOptionsProvider
     from ._lancedb import Session
 
+from .namespace_utils import (
+    _normalize_create_namespace_mode,
+    _normalize_drop_namespace_mode,
+    _normalize_drop_namespace_behavior,
+)
+
 
 class DBConnection(EnforceOverrides):
```

Comment:
Not related but it would be really nice to phase out these abstract classes :cold_sweat: